net/mscp/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                mscp
+version             0.2.1
+categories          net
+platforms           darwin
+license             GPL-3.0
+maintainers         @lasmarois \
```

Comment:
A port should be marked `openmainter` which allows for others to contribute updates to this port as well:

```
maintainers         {@lasmarois nicomarois.com:mail} \
                    openmaintainer
```